charts/invenio/templates/invenio-configmap.yaml

```diff
@@ -20,5 +20,8 @@ data:
   INVENIO_DATACITE_ENABLED: "False"
   INVENIO_LOGGING_CONSOLE_LEVEL: "WARNING"
   {{- range $key, $value := .Values.invenio.extra_config }}
-  {{ $key }}: {{ $value | toYaml | indent 4 | trim }}
+  {{ $key }}: {{ $value $ | toYaml | indent 5 | trim }}
+  {{- end }}
+  {{- range $key, $value := .Values.invenio.extraConfig }}
+  {{ $key }}: {{ tpl $value $ | toYaml | indent 5 | trim }}
```

Comment:
I have not tested this myself, so maybe it's obvious. But why `indent 5`? I think it's more common to keep YAML indentation at even multiples of 2 or 4. Am I missing something?

Reply:
Honestly, I don't know. It was probably me testing the template part ☺️ 
It does render correctly though ... or at least it renders the same output with both values ...
```
$ helm template -f test.yaml  ./charts/invenio -s templates/invenio-configmap.yaml
---
# Source: invenio/templates/invenio-configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: invenio-config
data:
  INVENIO_ACCOUNTS_SESSION_REDIS_URL: 'redis://....'
  INVENIO_APP_ALLOWED_HOSTS: '["test.com"]'
  ...
  INVENIO_CACHE_KEY_PREFIX: 'test::cache::'
  INVENIO_MAIL_DEFAULT_SENDER: no-reply@test.com
  ...
```

Reply:
I put it back to `4`, like it was originally.